<?xml version="1.0" encoding="UTF-8"?>

<!--
SPDX-FileCopyrightText: 2019-2023 Mattia Basaglia <dev@dragon.best>

SPDX-License-Identifier: GPL-3.0-or-later
-->

<ui version="4.0">
 <class>EmojiSetDialog</class>
 <widget class="QDialog" name="EmojiSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Emoji Set</string>
  </property>
  <property name="windowIcon">
   <iconset theme="face-smile">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="emoji_set_view">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>72</width>
       <height>72</height>
      </size>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>License</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Downloaded</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progress_bar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Reload</string>
       </property>
       <property name="icon">
        <iconset theme="view-refresh">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_view_website">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>View Website</string>
       </property>
       <property name="icon">
        <iconset theme="internet-web-browser">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_download">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Download</string>
       </property>
       <property name="icon">
        <iconset theme="download">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_add_emoji">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Add Emoji...</string>
       </property>
       <property name="icon">
        <iconset theme="smiley-add">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-close">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>EmojiSetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>751</x>
     <y>506</y>
    </hint>
    <hint type="destinationlabel">
     <x>788</x>
     <y>626</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>EmojiSetDialog</receiver>
   <slot>reload_sets()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_view_website</sender>
   <signal>clicked()</signal>
   <receiver>EmojiSetDialog</receiver>
   <slot>view_website()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>655</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>emoji_set_view</sender>
   <signal>currentCellChanged(int,int,int,int)</signal>
   <receiver>EmojiSetDialog</receiver>
   <slot>set_selected(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>633</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_download</sender>
   <signal>clicked()</signal>
   <receiver>EmojiSetDialog</receiver>
   <slot>download_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>688</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_add_emoji</sender>
   <signal>clicked()</signal>
   <receiver>EmojiSetDialog</receiver>
   <slot>add_emoji()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>563</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>reload_sets()</slot>
  <slot>view_website()</slot>
  <slot>set_selected(int)</slot>
  <slot>download_selected()</slot>
  <slot>add_emoji()</slot>
 </slots>
</ui>
